src/strategies/strategy-sorbetto-base.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+import "./strategy-base.sol";
+import "../interfaces/ISorbettoFragola.sol";
+
+abstract contract StrategySorbettoBase is StrategyBase {
+    address public token0;
+    address public token1;
+
+    // How much SUSHI tokens to keep?
+    uint256 public keepReward = 0;
+    uint256 public constant keepRewardMax = 10000;
+
+    constructor(
+        address _token0,
+        address _token1,
+        address _lp,
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyBase(
+            _lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {
+        token1 = _token1;
+        token0 = _token0;
+        IERC20(token0).safeApprove(_lp, uint(-1));
+        IERC20(token1).safeApprove(_lp, uint(-1));
+    }
+    
+    function balanceOfPool() public override view returns (uint256) {
+        return 0;
+    }
+
+    function getHarvestable() external view returns (uint256, uint256) {
+        (uint256 token0Rewards, uint256 token1Rewards,,) = ISorbettoFragola(want).userInfo(address(this));
+        return (token0Rewards, token1Rewards);
+    }
+
+    // **** Setters ****
+
+    function deposit() public override {}
```

Comment:
`deposit()` currently doesn't do anything. I believe you meant to include `ISorbettoFragola(want).deposit(_token0, _token1);`

Reply:
deposit or withdraw doesn't mean anything in PLP strategy, because we don't deposit or withdraw anything, just holding PLP will make yields for rebalance.